src/CodeGenerator/InjectorGenerator.php

```diff
@@ -185,6 +185,11 @@ public function getNamespace() : string
         return $this->namespace;
     }
 
+    public function getLogger() : LoggerInterface
```

Comment:
This is exposing the logger unnecessarily. As far as I can see this is only used in the Unit test suite.
Therfore we should mark this method either `@internal` or remove it and test the logger assignment differently.